cypress/integration/rendering/journey.spec.js

```diff
@@ -1,6 +1,6 @@
 import { imgSnapshotTest, renderGraph } from '../../helpers/util.ts';
 
-describe('User journey diagram', () => {
+describe('User journey diagram simple tests', () => {
```

Comment:
Please revert this change, this should fix part of argos test, some are shown as added/removed.